ui/component/publishName/view.jsx

```diff
@@ -1,51 +1,41 @@
 // @flow
```

Comment:
This was in the same component as PublishBid. I separated them to move the name field up to the top of the publish form